app/models/payment.rb

```diff
@@ -1,9 +1,17 @@
 class Payment < ApplicationRecord
-  belongs_to :claim
+  has_many :claims
   belongs_to :payroll_run
 
+  validates :claims, length: {maximum: 1}
```

Comment:
What situation in the running app code would this validation prevent?

Reply:
None; it served more as documentation than anything else. I've removed it.